### PR TITLE
feat(SNP-637): Update README.md

### PR DESCRIPTION
**Pull request by martynov-alex**
## Checklist for self-check
- [ ] Commits and PRs have been filed according to [the rules on the project](https://github.com/surfstudio/surf-flutter-app-template#workflow-in-a-repository).
- [ ] The author is marked as an assigne and assigned mandatory reviewers.
- [ ] Required labels marked
- [ ] Specified related tasks and/or related PRs.
- [ ] Specified Changes.
- [ ] Attached videos/screenshots demonstrating the fix/feature.
- [ ] All unspecified fields in the PR description deleted.
- [ ] New code covered by tests.

## Checklist for reviewers
- [ ] CI passed successfully _(with a green check mark)_.
- [ ] PR is atomic, by volume no more than 400 (+-) corrected lines (not including codogen).

Design:
- [ ] System design corresponds to the agreements on structure and architecture on the project.
- [ ] The code is decomposed into necessary and sufficient components.

Functionality:
- [ ] The code solves the problem.
- [ ] Any changes to the user interface are reasonable and look good.

Complexity:
- [ ] The code is clear, easy to read, functions are small, no more than 50 lines.
- [ ] The logic is not overcomplicated, there is no overengineering (no code sections that may be needed in the future, but no one knows about it).

Tests:
- [ ] Updated or added tests for mandatory components.
- [ ] The tests are correct, helpful, and well designed/developed.

Naming:
- [ ] The naming of variables, methods, classes and other components is understandable.

Comments:
- [ ] The comments are understandable and helpful.

Documentation:
- [ ] All labels are correct
- [ ] Technical documentation updated (after approval, updates last reviewer).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,430 +1,242 @@
...
+1. Specify the name of the application [here](#application-name).
+2. Search for `flutter_template` and replace it with the name of your project where needed.
+3. Configure Firebase with the [guide](docs/firebase.md).
+4. Initialize `FirebaseCrashlytics` (you can find that in `TODO(init-project)`).
+5. Install a needed Flutter version with FVM using the command:
+    ```sh
+    fvm use <flutter_version>
+    ```
+    If FVM is not installed, check [FVM workflow](#fvm-workflow).
+6. Specify the Flutter version used in the project [here](#flutter-and-dart-fvm-versions-of-the-project).
```
[the remark this answers is not in the document]
Comment (martynov-alex) [reply README.md:21]: I see this like:
For example we have Flutter 3.13.4 version now. But it doesn't matter why a teamlead wants 3.10.
So 
1. A teamlead "use" nedded version 
2. then writes the version down

File: README.md [hunk 2]
```diff
@@ -1,430 +1,242 @@
...
-}
-```
+Android Application Signature guide can be found [here](docs/sign_app_rules_android.md).
 
-### Theming
+iOS certs and profiles should be saved here: `ios/certs`.
 
-There are three layers of theme we use in our projects:
+> [!WARNING]
+> DO NOT STORE CERTIFICATES IN THE PROJECT REPOSITORY.
```
[the remark this answers is not in the document]
Comment (martynov-alex) [reply README.md:165]: wrote more clearly

File: README.md [hunk 3]
```diff
@@ -1,430 +1,242 @@
...
 
-#### Raw resources
+### CI/CD
 
-By raw resources we mean raw colors, fonts or assets. For example:
+> [!NOTE]
+> Add here CI/CD information or skip this section.
```
[the remark this answers is not in the document]
Comment (martynov-alex) [reply README.md:170]: Sadly i'm not an expert at CI/CD, but I've found the section here https://github.com/surfstudio/love-republic-flutter
I looks like you are right, will delete this

File: docs/sign_app_rules_android.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+# Application Signature
+
+To build a release version of the application, it is necessary to sign the apk file.
+
+To do this, follow these steps:
+
+- Create a keystore.
+- Set up the required Gradle tasks.
+- Upload the keystore to Jenkins.
```
[the remark this answers is not in the document]
Comment (martynov-alex) [reply docs/sign_app_rules_android.md:9]: fix

File: README.md [hunk 5]
```diff
@@ -1,430 +1,242 @@
...
+2. Search for `flutter_template` and replace it with the name of your project where needed.
+3. Configure Firebase with the [guide](docs/firebase.md).
+4. Initialize `FirebaseCrashlytics` (you can find that in `TODO(init-project)`).
+5. Install a needed Flutter version with FVM using the command:
+    ```sh
+    fvm use <flutter_version>
+    ```
+    If FVM is not installed, check [FVM workflow](#fvm-workflow).
+6. Specify the Flutter version used in the project [here](#flutter-and-dart-fvm-versions-of-the-project).
+7. Run this command to get up-to-date versions of dependencies:
```
[the remark this answers is not in the document]
Comment (martynov-alex) [reply README.md:22]: fix, move this point to app main README template

**Review by maxim-sysoev — APPROVED** [review]

**Pull request merged**